vignettes/getting_started.Rmd

```diff
@@ -0,0 +1,194 @@
+---
+title: "Setting up checklist"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Setting up checklist}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Starting an R package from scratch
+
+### Prepare online setup
+
+1. Go to https://github.com and login.
+  Create a new public repository in the INBO organisation.
+  Use the package name as repository name.
+  Enter the repository name and optionally a description.
+  Don't add a `README`, `.gitignore` or `LICENSE`.
+
```

Comment:
I would rather add this below, in 'Finalise online setup' below (where branch protection is set for the checks), and I would add it as a recommendation.  Personally I think status checks are more important, and they can suffice in case of small changes that do not influence the functioning of a package (e.g. typos).  In case of typos, the requirement for a review is often an overload (certainly for small packages with only 1 author).  On the other hand, for more important changes, a review is certainly required, but contributors normally ask it spontaneously in this situation...  So I think it is up to the maintainer to decide on this, but certainly a good idea to recommend it.

Reply:
The need for reviews depends on the situation. Reviews don't work well when you're the only one working on a package. Since we are instructing them to go to the branch protection page, they'll see the option to require reviews.